bot/commands/regulars.js

```diff
@@ -0,0 +1,50 @@
+// logs the regular users of the server
+
+const {Command} = require('yuuko');
+
+// TODO: first off this doesn't print everything if there's too many elements, also everything else in this sucks
+// also good chance this would freeze the bot for a long ass time in production
+module.exports = new Command('regulars', async (msg, args, context) => {
```

Comment:
> also good chance this would freeze the bot for a long ass time in production

User statistics is something I wanted to tackle later, probably as a separate process running in the background for heavy operations like this. This code could be useful, but we do want to avoid fetching 500k messages every time someone runs a command if we can help it - seems like a good way to make the ratelimit gods angry at us. :P

We can keep this code around for later, when I get time to work on user statistics/name tracking/etc later, but for now I think we should probably disable this command. If you rename the file to start with a period, the command framework will stop it from being loaded so it doesn't accidentally get used in a premature state.